templates/includes/content-block--bordered.html

```diff
@@ -0,0 +1,8 @@
+<div class="content-block--bordered">
+    <div class="container">
+        <div class="content-block__text">
+            <p class="mb-0">
```

Comment:
This is perhaps something for discussion but I'm not sure about this use of `mb-0` because we're bringing a bootstrap class in specifically to manage the spacing that should be the responsibility of the component. I'd personally opt for the `margin` to be specified within the Sass for the component rather than to have a dependency on Bootstrap.